index.md

```diff
@@ -15,7 +15,7 @@ plagued reflection-based solutions. More details can be found in
 
 ## Where are the docs?
 
-* [User documentation](users-guide)
+* [User documentation](users-guide.html)
```

Comment:
What was wrong with this? This appears to work, and our internal docs don't use `.html` endings, so it would be great if we didn't have to use MOE around this